client/src/containers/GameContainer.js

```diff
@@ -1,6 +1,6 @@
 import {useLoadScript} from "@react-google-maps/api"
 import { useState, useEffect } from "react";
-import Map from "../components/Map";
+import Map from "../components/map";
```

Comment:
Looks like the importing has been changed from Map to map, other than that it looks good!